src/ui/RealmBrowser/Content/Table/Table.tsx

```diff
@@ -162,14 +161,7 @@ export const Table = ({
         width={width}
       />
       {onAddColumnClick ? (
-        <AddColumnControl
-          isHidden={scrollLeft > 0 && scrollRight === 0}
```

Comment:
With this change the button will be overlapping the sort controls of the rightmost column (which might be intended?):

<img width="937" alt="add-property-position" src="https://user-images.githubusercontent.com/1243959/91957863-e3767780-ed06-11ea-8c76-b8dd48c233a9.png">

Reply:
I've tried implementing the margins set in the updated design, but ran into problems with the current grid and its absolute positioning.. If you have any quick solutions for this, I'm open for suggestions.

Reply:
Should we create another issue to track this?

Reply:
I've added https://github.com/realm/realm-studio/issues/1324